wasm/cli.js

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node --experimental-wasm-bigint
```

Comment:
This file seems like it should be in a different repo

Reply:
Absolutely. Some of the files (e.g. wasm/cli.js, package.json, README.md) just give an example how sassc.wasm can be run. I'll remove them before flipping this PR status to "Ready for review"

Reply:
Sounds good, wasn't sure if it was part of the scaffolding or not